packages/jsii/lib/compiler.ts

```diff
@@ -255,29 +260,22 @@ export class Compiler implements Emitter {
   private async buildTypeScriptConfig() {
     let references: string[] | undefined;
     let composite: boolean | undefined;
+    let incremental: boolean | undefined;
     if (this.projectReferences) {
       references = await this.findProjectReferences();
-      composite = true;
+      composite = incremental = true;
```

Comment:
```suggestion
      composite = true;
      incremental = true;
```
This feels like its gonna unnecessarily confuse people but maybe I'm just easily confused.

Reply:
Already done.